android/app/src/main/java/org/mozilla/magnet/magnetapi/ApiMagnet.java

```diff
@@ -17,5 +17,6 @@ public ApiMagnet(Context context) {
         mount("subscriptions", new ApiSubscriptions(context));
         mount("metadata", new ApiMetadata(context));
         mount("preferences", new ApiPreferences(context));
+        mount("tracking", new ApiAnalytics(context));
```

Comment:
The MagnetApi stuff was originally designed to be an interface to the `magnet-service` REST API. I don't think it should become the native interface for everything.

IMO we should expose the native tracking module via it's own dedicated JS API that matches the native API and keep the `api.*` namespace reserved for magnet-service HTTP calls.